app/models/applicant.rb

```diff
@@ -12,45 +12,11 @@ class Applicant < ApplicationRecord
   has_many :bank_providers
   has_many :bank_errors
 
-  validates :first_name, :last_name, :date_of_birth, :national_insurance_number, presence: true
-
-  validate :validate_date_of_birth, :validate_national_insurance_number
-
-  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }, allow_nil: true
-
   def email_address
     email
   end
 
   def full_name
     "#{first_name} #{last_name}".strip
   end
-
-  def dob
-    {
-      dob_year: date_of_birth.strftime('%Y'),
```

Comment:
we don't need this method anymore?

Reply:
It seems not. I had it in, and when the tests were run it was showing as no test coverage. I removed it, test coverage went to 100% and nothing broke.